assets/js/modules/analytics/common/profile-name.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Analytics Profile Name component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_FORMS } from '../../../googlesitekit/datastore/forms';
+import { TextField, HelperText, Input } from '../../../material-components';
+import { STORE_NAME, PROFILE_CREATE, FORM_SETUP } from '../datastore/constants';
+const { useSelect, useDispatch } = Data;
+
+export default function ProfileNameTextField() {
```

Comment:
The file name here should be the same as the name of the component (i.e. `ProfileNameTextField.js`). This is more of a recent change so it isn't exactly obvious yet. We have an issue to update all the current usage separately. See #1654

Reply:
Updated.